Header header

uint64   index  #index of Waypoint in current WaypointList to remove
